pandas-stubs/core/frame.pyi

```diff
@@ -676,6 +676,13 @@ class DataFrame(NDFrame, OpsMixin, _GetItemHack):
         self,
         expr: _str,
         *,
+        parser: Literal["pandas", "python"] = ...,
+        engine: Literal["python", "numexpr"] | None = ...,
+        local_dict: dict[_str, Any] | None = ...,
+        global_dict: dict[_str, Any] | None = ...,
+        resolvers: list[Mapping] | None = ...,
+        level: int = ...,
+        target: object | None = ...,
```

Comment:
Can you indicate where you found this list of possible keyword arguments?

Reply:
I sourced them by tracing the execution to:
https://github.com/pandas-dev/pandas/blob/337d40e5d55f7787e48f029486f47fd5a053bc80/pandas/core/computation/eval.py#L177-L187